### PR TITLE
Fix confirm send screen amount display

### PR DESCRIPTION
**Pull request by PanRadaCloudbest**
## What does this PR do?

Fix confirm send screen amount display

## Todos:

- [ ] Checked on iOS
- [ ] Checked on Android

### CODE REVIEW

File: src/screens/SendCoinsConfirmScreen.tsx [hunk 1]
```diff
@@ -211,8 +211,7 @@ class SendCoinsConfirmScreen extends Component<Props> {
           <View>
             <View style={styles.chooseWalletButton}>
               <Text style={typography.headline4}>
-                {roundBtcToSatoshis(item.amount) || satoshiToBtc(item.value).toString()}{' '}
-                {fromWallet.preferredBalanceUnit}
+                {item.amount.toFixed(8) || satoshiToBtc(item.value).toString()} {fromWallet.preferredBalanceUnit}
```
Comment (Kamill90) [inline src/screens/SendCoinsConfirmScreen.tsx:214]: don't we need that ugly space between value and unit?

Comment (PanRadaCloudbest) [reply src/screens/SendCoinsConfirmScreen.tsx:214]: true

Comment (PanRadaCloudbest) [reply src/screens/SendCoinsConfirmScreen.tsx:214]: but the problem is my Vscode removed this 
have clue how manage that?

Comment (PanRadaCloudbest) [reply src/screens/SendCoinsConfirmScreen.tsx:214]: done

**Review by Kamill90 — APPROVED** [review]

**Pull request merged**